torch/onnx/_internal/jit_utils.py

```diff
@@ -11,8 +11,15 @@
 import torch
 from torch import _C
 from torch._C import _onnx as _C_onnx
+from torch.onnx import errors
 from torch.onnx._globals import GLOBALS
-from torch.onnx._internal import _beartype
+from torch.onnx._internal import _beartype, registration
+
+# TODO(titaiwang): remove this when onnx-script becomes dependency
+try:
+    import onnxscript  # type: ignore[import]
+except Exception:
+    raise errors.OnnxExporterError("Module onnxscript is not installed!")
```

Comment:
We should prob just do nothing if onnxscript is not found

Reply:
I removed the whole thing, since it was only for annotation.